src/index.html

```diff
@@ -17,6 +17,11 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="stopwatch__hand">
+        <div class="stopwatch__minutes-hand speed-up-min"></div>
+        <div class="stopwatch__seconds-hand speed-up-sec"></div>
```

Comment:
The BEM modifier classes `speed-up-min` and `speed-up-sec` should be used for testing purposes only. They should be removable without affecting the basic functionality of the stopwatch, as per the task requirements.